INSTALL.md

```diff
@@ -43,11 +43,19 @@ The repeated www.naturalearthdata.com in the Natural Earth shapefiles is correct
 Put these shapefiles at `path/to/openstreetmap-carto/data`.
 
 ## Fonts
-The stylesheet uses Noto Sans, an openly licensed font from Google with support for multiple scripts. The "UI" version is used where available, with its vertical metrics which fit better with Latin text. Other fonts from the Noto family are used for some other languages.
+The stylesheet uses Noto, an openly licensed font from Google with support for multiple scripts. The “UI” version is used where available, with its vertical metrics which fit better with Latin text. The stylesheet uses Noto’s “Sans” style where available. If not available, this stylesheet uses another appropriate style of the Noto family.
```

Comment:
When referring to "Noto" instead of "Noto Sans" it should be called a "font family" instead of a "font". The information about Sans should come before the information about UI, as UI is a finer division than Sans.